bigframes/_config/compute_options.py

```diff
@@ -97,7 +103,9 @@ class ComputeOptions:
     semantic_ops_threshold_autofail = False
 
     ai_ops_confirmation_threshold: Optional[int] = 0
-    ai_ops_threshold_autofail = False
+    ai_ops_threshold_autofail: bool = False
+
+    allow_large_results: bool = False
```

Comment:
Please make this an `Optional[bool]` so that we can disambiguate when someone sets it. This should take precedence over the bigquery option, not replace it. It's too late to make a breaking change.

```suggestion
    allow_large_results: Optional[bool] = None
```